registry/file/backend.go

```diff
@@ -1,30 +1,133 @@
 // Package file implements a simple file based registry
-// backend which reads the routes from a file once.
+// backend which reads the routes from a file.
+//file content like registry.static.routes = route add web-svc /test http://127.0.0.1:8082
+//registry.file.path = /home/zjj/fabio.txt
+//registry.file.noroutehtmlpath = /home/zjj/404.html
 package file
 
 import (
 	"io/ioutil"
 	"log"
+	"os"
+	"time"
 
 	"github.com/fabiolb/fabio/config"
 	"github.com/fabiolb/fabio/registry"
-	"github.com/fabiolb/fabio/registry/static"
+)
+
+type be struct {
+	cfg          *config.File
+	routeMtime   time.Time
+	norouteMtime time.Time
+	Routes       string
+	NoRouteHTML  string
+	Interval     time.Duration
+}
+
+var (
+	zero time.Time
 )
 
 func NewBackend(cfg *config.File) (registry.Backend, error) {
-	routes, err := ioutil.ReadFile(cfg.RoutesPath)
-	if err != nil {
-		log.Println("[ERROR] Cannot read routes from ", cfg.RoutesPath)
+	b := &be{cfg: cfg, Interval: 2 * time.Second}
```

Comment:
Can you make the `Interval` configurable? See https://github.com/fabiolb/fabio/blob/master/config/load.go#L161 for an example.

Please also update the website documentation at

https://github.com/fabiolb/fabio/tree/master/docs/content/ref

Reply:
update to commited.